universal-application-tool-0.0.1/app/views/dev/FileUploadViewStorageStrategy.java

```diff
@@ -0,0 +1,23 @@
+package views.dev;
+
+import j2html.tags.ContainerTag;
+import services.cloud.StorageUploadRequest;
+import services.cloud.aws.SignedS3UploadRequest;
+import services.cloud.azure.BlobStorageUploadRequest;
+import views.HtmlBundle;
+import views.ViewUtils;
+
+/**
+ * Interface for interacting with different cloud storage providers for the file upload view. We
+ * currently support Azure and AWS.
```

Comment:
```suggestion
 * Interface for interacting with different cloud storage providers for the file upload view.
```
So this doesn't need to be updated when we add GCP